opcua.cpp

```diff
@@ -955,6 +956,40 @@ void OPCUA::parseConfig(ConfigCategory &config)
 	{
 		m_dcfDeadbandValue = std::stod(config.getValue("dcfDeadbandValue"));
 	}
+
+	/**
+	 * Updates the allowed control nodes list from the configuration.
+	 *
+	 * Clears the current allowed control nodes, retrieves the node list from the configuration,
+	 * and logs the updated allowed control nodes.
+	 */
+	if (config.itemExists("controlList"))
+	{
+		// Clear the current list of allowed control nodes
+		m_allowedControlNodes.clear();
+        m_nodeBuiltinIdCache.clear();
```

Comment:
Good catch by Nandan. The indentation of this line is done with spaces, however. The rest of the code is indented with tabs.